tests/test_github.py

```diff
@@ -594,6 +598,146 @@ def test_github_create_pr_missing_base_branch_on_remote(self) -> None:
             """,
         )
 
+    git_api_state_for_test_github_create_pr_with_multiple_non_origin_remotes = MockGitHubAPIState(
+        [
+            {
+                'head': {'ref': 'branch-1', 'repo': mock_repository_info},
+                'user': {'login': 'github_user'},
+                'base': {'ref': 'root'}, 'number': '15',
+                'html_url': 'www.github.com', 'state': 'open'
+            }
+        ],
+        issues=[
+            {'number': '16'},
+            {'number': '17'},
+            {'number': '18'},
+            {'number': '19'},
+        ]
+    )
+
+    @mock.patch('git_machete.cli.exit_script', mock_exit_script)
+    @mock.patch('git_machete.client.MacheteClient.ask_if', mock_ask_if)
+    # We need to mock GITHUB_REMOTE_PATTERNS in the tests for `test_github_create_pr` due to `git fetch` executed by `create-pr` subcommand.
+    @mock.patch('git_machete.github.GITHUB_REMOTE_PATTERNS', FAKE_GITHUB_REMOTE_PATTERNS)
+    @mock.patch('git_machete.utils.run_cmd', mock_run_cmd)  # to hide git outputs in tests
+    @mock.patch('git_machete.options.CommandLineOptions', FakeCommandLineOptions)
+    @mock.patch('urllib.error.HTTPError', MockHTTPError)  # need to provide read() method, which does not actually reads error from url
+    @mock.patch('urllib.request.Request', git_api_state_for_test_github_create_pr_with_multiple_non_origin_remotes.new_request())
+    @mock.patch('urllib.request.urlopen', MockContextManager)
+    @mock.patch('builtins.input', mock_input)
```

Comment:
Wow I didn't know the package is called `builtins` for stuff like `input` 😯

Reply:
Me neither until now :D